src/wasm/literal.cpp

```diff
@@ -164,6 +165,7 @@ std::ostream& operator<<(std::ostream& o, Literal literal) {
     case Type::i64: o << literal.i64; break;
     case Type::f32: literal.printFloat(o, literal.getf32()); break;
     case Type::f64: literal.printDouble(o, literal.getf64()); break;
+    case Type::v128: assert(false && "v128 not implemented yet");
```

Comment:
pretty crucial for debugging, so I'd make this high priority - this PR or quickly after.